agent/lib/kontena/workers/container_log_worker.rb

```diff
@@ -67,7 +68,7 @@ def on_message(id, stream, chunk)
         type: stream,
         data: data
       }
-      @queue << msg
+      @queue << msg if @queue.size < QUEUE_LIMIT
```

Comment:
Do we really want to log every time we discard a log item? It will exhaust agent logs, yes?